EdFi.Ods.Utilities.Migration/logistics/scripts/modules/load-path-resolver.ps1

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+param(
+    [string[]] $repositoryNames
+)
+
+if ((Get-Module | Where-Object -Property Name -eq 'path-resolver')) { return }
+
+Import-Module -Force -Scope Global "$PSScriptRoot\path-resolver.psm1" -ArgumentList @(, $repositoryNames)
```

Comment:
again not sure want to use path resolver.. any reason to?

Reply:
yes ,  I need it for $PackageFiles = @(
            @{ nuspecsrc = (Get-RepositoryResolvedPath "../../Ed-Fi-MigrationUtility/EdFi.Ods.Utilities.Migration/EdFi.Standard.Descriptors/");src = (Get-RepositoryResolvedPath "../../Ed-Fi-Standard/v3.1/Descriptors/"); target = (Join-Path $OutputPath "any/netcoreapp3.1/Descriptors/3.1") }
        )